### PR TITLE
specs: Document starting timestamp and block num vars

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

Adds words to the specs and deploy-config.ts file to clarify the intent of the L2OO starting timestamp and block number config variables

### CODE REVIEW

**Review by trianglesphere — APPROVED** [review]
thanks for clarifying the specs

File: specs/proposals.md [hunk 1]
```diff
@@ -96,12 +96,23 @@ The exact number is yet to be determined, and will depend on the design of the f
 
 The L2 Output Oracle contract implements the following interface:
 
-```js
+```solidity
```
Comment (tynes) [inline specs/proposals.md:99]: I am against inlining solidity in the specs. We should link to the solidity and have pythonic pseudocode when we need code. Its too easy for things to get out of date

**Comment by tynes** [comment]
I do think this is a good change

File: specs/proposals.md (hunk 1 above)
Comment (maurelian) [reply specs/proposals.md:99]: I agree, but I just wanted to improve the existing specs. That change is another PR touching several docs.

File: specs/proposals.md [hunk 2]
```diff
@@ -42,10 +43,11 @@ described [below](#l2-output-commitment-construction).
 
 If there is no newly finalized output, the service continues querying until it receives one. It then submits this
 output, and the appropriate timestamp, to the [L2 Output Root](#l2-output-root-smart-contract) contract's
-`proposeL2Output()` function. The timestamp MUST be the next multiple of the `SUBMISSION_INTERVAL` value.
+`proposeL2Output()` function. The timestamp block number must correspond to the `startingBlockNumber` plus the next
+multiple of the `SUBMISSION_INTERVAL` value.
 
-The proposer may also delete the most recent output root by calling the `deleteL2Output()` function.
-The function can be called repeatedly if it is necessary to roll back the state further.
+The proposer may also delete multiple output root by calling the `deleteL2Output()` function and specifying the index of
```
Comment (norswap) [inline specs/proposals.md:49]:
```suggestion
The proposer may also delete multiple output roots by calling the `deleteL2Output()` function and specifying the index of
```

**Review by norswap — APPROVED** [review]

File: specs/proposals.md (hunk 2 above)
Comment (maurelian) [reply specs/proposals.md]: fixed thanks!

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**